DigitalLearningSolutions.Data/DataServices/GroupsDataService.cs

```diff
@@ -180,5 +182,28 @@ public void DeleteGroupDelegatesRecordForDelegate(int groupId, int delegateId)
                 new {groupId, delegateId}
             );
         }
+
+        public int AddDelegateGroup(int centreId, string groupLabel, string? groupDescription, int adminUserId)
+        {
+            var values = new
+            {
+                centreId,
+                groupLabel,
+                groupDescription,
+                linkedToField = 0,
+                syncFieldChanges = 0,
+                addNewRegistrants = 0,
+                populateExisting = 0,
+                createdDate = DateTime.UtcNow,

```

Comment:
This can either live in the SQL query (as it should never be anything other than UTC now), or if we want to specify it in the C# code, we should use clock service for easy mocking in unit tests.